book/06-github/sections/5-scripting.asc

```diff
@@ -147,7 +117,7 @@ $ curl https://api.github.com/users/schacon
 }
 ----
 
-There are tons of endpoints like this to get information about organizations, projects, issues, commits -- just about anything you can publicly see on GitHub. You can even use the API to render arbitrary Markdown or find a `.gitignore` template.
+Il y a des tas de points d'entrée comme celui-ci pour obtenir des informations sur des regroupements, projets, problèmes, _commits_ -- en fait n'importe quelle chose que vous pouvez voir publiquement sur GitHub. Vous pouvez même utiliser l'interface de programmation pour interpréter du Markdown arbitraire ou trouver un modèle `.gitignore`.
```

Comment:
Pareil, j'ai cherché pour comprendre le sens de "render" ici. Pourquoi est-ce que l'auteur fait des phrases compliquées ? Pourquoi n'a-t-il pas écrit plutôt "to write text as Markdown" ou quelque chose dans le genre ?
J'ai cherché à retranscrire l'idée de l'auteur pour rester le plus possible fidèle au texte d'origine.